Comments/Services/CommentService.cs

```diff
@@ -230,10 +230,28 @@ public CommentsAndQuestions FilterCommentsAndQuestions(CommentsAndQuestions comm
 			    commentsAndQuestions.Questions.ForEach(q => q.Show = (!q.Show || !q.DocumentId.HasValue) ? false : documentIds.Contains(q.DocumentId.Value));
 			    commentsAndQuestions.Comments.ForEach(c => c.Show = (!c.Show || !c.DocumentId.HasValue) ? false : documentIds.Contains(c.DocumentId.Value));
 			}
+
+			var commenters = commentersToFilter?.ToList() ?? new List<string>(); ;
+			if (commenters.Any())
+			{
+				commentsAndQuestions.Comments.ForEach(c => c.Show = (!c.Show || c.CommenterEmail.Length == 0) ? false : commenters.Contains(c.CommenterEmail));
```

Comment:
is the email ever lowercased? if so the contains needs to support  that. 

however if we store the email address and display it again in the same case it was input this is fine.

Reply:
Added StringComparer.OrdinalIgnoreCase to the contains